doc/hotspot-style.html

```diff
@@ -207,23 +207,31 @@ <h3 id="source-files">Source Files</h3>
 <ul>
 <li><p>All source files must have a globally unique basename. The build
 system depends on this uniqueness.</p></li>
+<li><p>Keep the include lines within a section alphabetically sorted.</p></li>
+<li><p>Put conditional inclusions (`#if ...`) at the end of the section of HotSpot
+include lines. This also applies to macro-expanded includes of platform
+dependent files.</p></li>
```

Comment:
What is the order for the conditional sections? Alphabetic on the include guard?

Reply:
I don't think there's a set order. I wouldn't mind making it alphabetic with platforms includes coming before the other conditional includes.

Reply:
FWIW, I also tend to sort the forward declarations but that's also not something that everyone does.

Reply:
> FWIW, I also tend to sort the forward declarations but that's also not something that everyone does.

I, too, am an obsessive sorter. Clean code is good code.